### ISSUE TITLE
More accurate demo?

### ISSUE THREAD

**Issue by HaiDang2001VN** [mattpoggi/pydnet#2]
Hi @mattpoggi ,
Thank you and your partners so much for releasing this magnificent network in order to addressing the embedded case of Depth Estimation Problem.
I can see that you decided to run demo directly on webcam, but your inputs in this case is indoor type usually, whereas your checkpoint is trained on KITTI dataset (or pretrained on CS and fine-tuned on KITTI as in your paper). So it produces lower accuracy when captures images from webcam (but outputs still reasonable at all 👍 ), although run very fast, impressively 2Hz on my machine (my Intel i5 without NVidia GPU makes most of other networks take above 7-8s per image).
Therefore, if you want this network makes more satisfactory results when demo, it will be necessary to train it on indoor dataset (as NYUv2 or TUM RGB-D dataset), but it seems to be in the next year as IROS'18 is coming in the next 2 months.
Thanks in advance.

**Comment by mattpoggi** [comment]
Hi @HaiDang2001VN ,
thanks for your kind words. I agree with you that using a CS+K checkpoint indoor results in poor accuracy. We made available this code to provide a plug-and-play demo of our model, predicting depth on images acquired from your webcam without requiring any dataset (e.g., KITTI) to be downloaded. The main purpose is to show runtime, we included the CS+K checkpoint to have some prettier visualization compared to random predictions. 
We are planning to train on a custom indoor stereo dataset, we will release a checkpoint for that when ready.
